<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PipPage</class>
 <widget class="QWidget" name="PipPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>389</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure pip&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line9_3">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Index URL</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="indexEdit">
        <property name="toolTip">
         <string>Enter the URL of the package index or leave empty to use the default</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="indexLabel">
        <property name="text">
         <string/>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>pip Executables</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="7">
       <widget class="E5ListView" name="stringList">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="addButton">
        <property name="toolTip">
         <string>Press to add an entry</string>
        </property>
        <property name="text">
         <string>&amp;Add...</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="removeButton">
        <property name="toolTip">
         <string>Press to remove the selected entries</string>
        </property>
        <property name="text">
         <string>&amp;Remove</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="removeAllButton">
        <property name="toolTip">
         <string>Press to remove all entries</string>
        </property>
        <property name="text">
         <string>R&amp;emove All</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="defaultListButton">
        <property name="toolTip">
         <string>Press to load the default list</string>
        </property>
        <property name="text">
         <string>&amp;Default</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>196</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5ListView</class>
   <extends>QListView</extends>
   <header>E5Gui/E5ListView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>indexEdit</tabstop>
  <tabstop>stringList</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>removeAllButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
